public/dom/index.js

```diff
@@ -5,13 +5,15 @@
 
 function clientRequest(method, url, body, callback) {
   var xhr = new XMLHttpRequest();
-  xhr.addEventListener('load', function () {
-    if (xhr.readyState === 4 && xhr.status === 200) {
-      callback(JSON.parse(xhr.responseText));
-    } else {
-      console.log('XHR error', xhr.status);
-    }
-  });
+  if (method.toLowerCase() !== 'delete') {
+    xhr.addEventListener('load', function () {
+      if (xhr.readyState === 4 && xhr.status === 200) {
+        callback(JSON.parse(xhr.responseText));
+      } else {
+        console.log('XHR error', xhr.status);
```

Comment:
Need to handle this error so that it is displaying to the user - added to refactor issue #59